uint32 base_pwm  
uint32 stick_servo
uint32 boom_servo
uint32 pump_pwm
